src/blockchain_utilities/blockchain_export.cpp

```diff
@@ -178,7 +178,7 @@ int main(int argc, char* argv[])
   }
   r = core_storage->init(db, opt_testnet);
 
-  CHECK_AND_ASSERT_MES(r, false, "Failed to initialize source blockchain storage");
+  CHECK_AND_ASSERT_MES(r, 1, "Failed to initialize source blockchain storage");
```

Comment:
Notice: this is more than a non-functional change. Previously 0 (false) would have been returned from main() on error. The correct behavior, as in surrounding code is to return 1 on error.